.github/workflows/linters.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   linters:
     runs-on: ubuntu-20.04
-    continue-on-error: false
+    continue-on-error: true
```

Comment:
What does it mean🤔

Reply:
https://github.com/DimensionDev/Maskbook/runs/1569259125?check_suite_focus=true

more problems can be found

e.q: code style and type error